pkg/kubectl/cmd/attach.go

```diff
@@ -76,7 +82,7 @@ func NewCmdAttach(f cmdutil.Factory, cmdIn io.Reader, cmdOut, cmdErr io.Writer)
 			cmdutil.CheckErr(options.Run())
 		},
 	}
-	// TODO support UID
+	cmd.Flags().DurationVar(&options.GetPodTimeOut, "pod-running-timeout", defaultPodAttachTimeout, "The length of time (like 5s, 2m, or 3h, higher than zero) to wait until at least one pod is running")
```

Comment:
I wanted this being a helper function, as well. This will make sure we have the same name everywhere.